### PR TITLE
inputField fix

### PR DESCRIPTION
**Pull request by nichoop**

### CODE REVIEW

File: src/components/Form.js [hunk 1]
```diff
@@ -1,21 +1,22 @@
 import { useRouter } from "next/router";
+import Input from "@/src/components/Input.js";
```
Comment (niehaus-alexander) [inline src/components/Form.js:2]: You should use "Styled" as a prefix for Styled components that you import from your styles component folder. Additionally you have a Button component in your styles folder. Components do not belong in the styles folder. Only put styled components into the styles folder and follow the naming convention: "StyledButton".

File: src/components/Form.js [hunk 2]
```diff
@@ -94,19 +130,25 @@ export default function Form({
...
-      <label htmlFor="description">Description</label>
-      <br />
-      <textarea id="description" name="description" rows="5" cols="50" placeholder="Add a description (optional)"
-                defaultValue={initialData.description} />
+      <br />{" "}
+      <Input
+        label="Description:"
+        name="description"
+        placeholder="Enter your description here..."
+      />
```
Comment (niehaus-alexander) [inline src/components/Form.js:151]: According to your AC's the description field should be filled with the transactions description in edit mode. This is not the case, the description field is empty. Declare a defaultValue for the description field.

File: src/components/Input.js [hunk 3]
```diff
@@ -1,9 +1,5 @@
 import { useState } from "react";
-import {
-  InputContainer,
-  Label,
-  Textarea,
-} from "/src/components/styles/InputStyle.js";
+import { Label, Textarea } from "/src/components/styles/InputField.js";
```
Comment (niehaus-alexander) [inline src/components/Input.js:2]: Same here, use the prefix "Styled" and also your whole styles folder structure is pretty messy, you import Label and Textarea from a file called InputField.js. Create a separate file for each styled component that you reuse. Right now your your use of styledComponents is inconsistent and hard to understand. Checkout the handouts for styled components I and II.

File: src/components/styles/InputField.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import styled from "styled-components";
```
Comment (niehaus-alexander) [inline src/components/styles/InputField.js:1]: this whole file makes no sense to me. You name it InpuField.js but there is no StyledInput, but a label and textarea...

**Review by niehaus-alexander — CHANGES_REQUESTED** [review]
Check how to use and correctly apply styled Components again.

File: src/components/Form.js (hunk 1 above)
Comment (nichoop) [reply src/components/Form.js:2]: Will be resolved in later issue.

File: src/components/styles/InputField.js (hunk 4 above)
Comment (SabinePfanner) [reply src/components/styles/InputField.js:1]: I do not see, that this comment is resolved, as the file has not been changed. I agree that you could improve your naming for better readability for externals - but up to you

**Review by SabinePfanner — COMMENTED** [review]
I cannot see that the recommendations from Alex have been applied nor any other changes - therefore I guess could have been skipped, if you do not want to change the naming etc.

**Pull request merged**